README.md

```diff
@@ -49,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents].delete("Belle")
+town[:castle][:guests].push("Belle")
 ```
```

Comment:
This is fine, and it works. But there's something fishy about it. Is the thing that you are deleting and pushing the same object?